exporter/azure_monitor/creator/creator.go

```diff
@@ -0,0 +1,19 @@
+package creator
+// Package: Function used to create exporters
+
+import (
+	"log"
+	
+	"go.opencensus.io/exporter/azure_monitor"
+	"go.opencensus.io/trace"
+)
+
+/* Create and set exporter for Azure Monitor */
+func EnableObservabilityAndExporter() {

```

Comment:
What is the purpose of this function?

Reply:
If user's call this instead, they will not be able to specify an iKey, is this what you want?

Reply:
I wanted to reduce redundant code. This function creates an exporter and registers it for the trace to recognize it. If a developer forgets to add `trace.RegisterExporter(exporter)` then no data would be exported to Azure Monitor. It's not much code, so I can just have the function in each files that uses it. 

Good point. I intended this file only to be used for my testing purposes. It can be confusing for the user, so I will delete this file. Also I've realized I have been saying user and developer interchangeably. I will start to call them users.